front/app/components/admin/GraphCards/RegistrationsCard/useRegistrations/parse.ts

```diff
@@ -37,7 +36,7 @@ const getDate = (row: TimeSeriesResponseRow) => {
 const _parseTimeSeries = timeSeriesParser(getDate, parseRow);
 
 export const parseTimeSeries = (
-  responseTimeSeries: Response['data']['attributes'][0],
+  responseTimeSeries: RegistrationsResponse['data']['attributes'][0],
   startAtMoment: Moment | null | undefined,
   endAtMoment: Moment | null,
```

Comment:
They're always `Moment`s when they get passed to the `parseTimeSeries` function, because they we need to do some date math with them. Or is that not what you meant?

Reply:
I was just wondering why `startAtMoment` can be undefined and not `endAtMoment`.

Reply:
Yeah this is just some annoying inconsistency that's coming all the way from the way we implemented the date filters in the dashboard. Would love to one day fix this but out of scope for now